### ISSUE TITLE
publish daily workflow and release process

### ISSUE THREAD

**Issue by sarob** [opendatainitiative/opendatainitiative.github.io#15]
Manual process (this is easy while the number of contributors is low)
1. There are three active projects. opendatalegislation and transparency project data is integrated in the opendatainitiative.github.io project that creates the [https://opendatainitiative.github.io/](https://opendatainitiative.github.io/) website.
1. Each friday, each active project increments it's active branch revision number by 0.X.0 (tenth)
1. If your changes are small and need to pushed before the end of the week, use branch revision number incremented by 0.0.Y (hundredth)
1. Each contributor synchronizes their local repositories with the remote before starting work
1. Review the issues. Bugs are highest priority and may be quick work. Enhancements may take days to weeks.
1. Work the changes locally. Keep your work to a single problem or issue 
1. Verify the changes fixes problems and doesn't create new problems by running Jekyll locally. If you are working on the opendatalegislation or transparency project branch, you will need to copy your changes into the opendatainitiateve.github.io local repository to verify functionality. Make sure your changes are not in the master branch, rather in the next revision branch.
1. When the branch changes are complete, push the branch to the repository. The commit message needs to include the issue (if any) that it closes or updates, e.g. "closes bug Correct CA legislation review first line to not public #35". 
1. If the contributor is a member of the Core Contributor team, the pushed changes can be merged with the current branch immediately.
1. For everyone else, they need to create a Pull Request. One or more of the Core Contributor team member will review the change before the end of the week. Depending on the change, it may get merged with the master branch, request changes to the commit, request more explanation, delayed to a future revision, or declined.
1. When a Core Contributor merges pushed changes to a branch, any referenced issues will be updated and closed if appropriate. 
1. When a branch is complete, a Core Contributor will take the branch and merge the revision branch with the master branch.
1. During the week, a Core Contributor can decide a 0.0.Y hundredth branch revision needs to be published of in any of the opendatainitiative projects
1. At the end of the business week (Friday), a Core Contributor will take the latest merged branch 0.X.Y, 0.J.K revisions of the opendatalegislation and transparency project data, copy the files into the branch 0.B.C revision of the opendatainitiative.github.io project. This consists of matching the changes in each project since the last merged branch to the current branch. Each of these files gets copied into the opendatainitiative.github.io project. The resulting local website will be double checked for errors. If the site fails to load or has other serious errors, the opendatainitiative.github.io project branch 0.B.C revision can be delayed until the end of the week (Sunday). If the end of the week opendatainitiative.github.io project branch 0.B.C revision is missed, then the revision is postponed until the following end of the business week. 
1. For the opendatainitiative.github.io project branch 0.B.C commit message, use each of the project branch URL
1. After the all the project branches are updated, now the master branches need to be merged with the each of the latest branches. Once completed, each project latest branch and master must be pointing at the same, most recent change.
1. Create the next revision branch in each project and change the focus to that branch, so your local changes always land in the correct place.

Future
1. Code part or all of these steps to eliminate errors, speed development time, and allow more time to create unit and integration tests
1. Continue to allow local CI toolchain while running the CI toolchain remotely. Each push would get some basic tests run, results logged, and published for core contributors to review.
